### ISSUE TITLE
Error: Cannot read propriety 'delay' of undefined

### ISSUE THREAD

**Issue by somerandomcloud** [link-discord/crystalbot#1]
Hello! I was curious to testing this project out. When I start it everything works well, but when I try to run the command crystalaura, my console gets spammed with this error:
```
TypeError: Cannot read property 'delay' of undefined
    at AutoCrystal.<anonymous> (/home/host/Downloads/crystalbot-main/node_modules/mineflayer-autocrystal/lib/AutoCrystal.js:159:66)
    at Generator.next (<anonymous>)
    at fulfilled (/home/host/Downloads/crystalbot-main/node_modules/mineflayer-autocrystal/lib/AutoCrystal.js:5:58)```
Do you know how I can fix this issue? I followed the readme to the best of my abilities

**Comment by link-discord** [comment]
thats an issue with the plugin itself
I will look into it

**Comment by link-discord** [comment]
This should be fixed in the latest commit

**Comment by link-discord** [comment]
Please let me know if it works

**Comment by somerandomcloud** [comment]
Yeah ill test it rn

**Comment by somerandomcloud** [comment]
So it did fix the original problem, but a new issue appeared. As before, bot ran well just before starting crystalaura. My console spams this:
```
TypeError: Cannot read property 'value' of undefined
    at getAttributeValue (/home/host/crystalbot/node_modules/mineflayer/lib/plugins/explosion.js:41:16)
    at EventEmitter.bot.getExplosionDamages (/home/host/crystalbot/node_modules/mineflayer/lib/plugins/explosion.js:81:21)
    at AutoCrystal.<anonymous> (/home/host/crystalbot/node_modules/mineflayer-autocrystal/lib/AutoCrystal.js:61:37)
    at Generator.next (<anonymous>)
    at /home/host/crystalbot/node_modules/mineflayer-autocrystal/lib/AutoCrystal.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/home/host/crystalbot/node_modules/mineflayer-autocrystal/lib/AutoCrystal.js:4:12)
    at AutoCrystal.placeCrystal (/home/host/crystalbot/node_modules/mineflayer-autocrystal/lib/AutoCrystal.js:43:16)
    at AutoCrystal.<anonymous> (/home/host/crystalbot/node_modules/mineflayer-autocrystal/lib/AutoCrystal.js:159:36)
    at Generator.next (<anonymous>)
```

**Comment by link-discord** [comment]
This should also be fixed soon

**Comment by somerandomcloud** [comment]
Goodie, thank you!

**Comment by link-discord** [comment]
Also incase ur wondering that error comes from mineflayer directly not from my code or my auto-crystal plugin.
It happens when the auto-crystal plugin tries to calculate the damage but your bot doesn't have armor on.

**Comment by link-discord** [comment]
Should be fixed in the latest commits. Try downloading the new code and try using the bot again.

**Comment by somerandomcloud** [comment]
Oh it should have armor on?

**Comment by Robthekilla** [comment]
Can you see my issue too
